Tasks/XamarinTestCloudV1/XamarinTestCloud.ps1

```diff
@@ -193,4 +193,11 @@ if($testCloudResults)
     Write-Host "##vso[task.addattachment type=Distributedtask.Core.Summary;name=Xamarin Test Cloud Results;]$mdReportFile"
 }
 
+$featureFlags = @{
+    failDeprecatedBuildTask  = [System.Convert]::ToBoolean($env:FAIL_DEPRECATED_BUILD_TASK)
+}
+if ($featureFlags.failDeprecatedBuildTask)
+{
+	throw "The XamarinTestCloud@1 (Xamarin Test Cloud) task has been deprecated since January 11, 2018 and will soon be retired. Use the AppCenterDistribute@3 task instead."
```

Comment:
Shouldn't this string be taken from https://github.com/microsoft/azure-pipelines-tasks/blob/d4e470b1867696b700121b4170a2e1f2cbe2a077/Tasks/XamarinTestCloudV1/task.json#L30?
Now it is duplicated

Reply:
Same reason as here https://github.com/microsoft/azure-pipelines-tasks/pull/19471#discussion_r1467540461